docker-run-generate-outputs.sh

```diff
@@ -84,30 +84,32 @@ fi
 docker start -a -i "$container"
 
 # Copy the output data back out of the container
+
+if [[ "$PROFILE_CPU" = true ]]; then
```

Comment:
Why is this moving around? If the pipeline succeeded, all these files should be available and be copied out. If the pipeline didn't, then nothing will be copied. I think that's what we want; if we break the atomicity here we need to scream very loudly on the console to avoid having inconsistent data on the host's output folder.

Reply:
(Following that logic, the order here shouldn't matter, so still happy to approve).

Reply:
Currently If the pipeline quits at the advanced stages due to memory or cpu issues we don't copy out the logs. Therefore no file to investigate.

Reply:
That's still true here.

Reply:
Ah right. Updated thanks!